src/main/app/src/static/smart-documents-result.html

```diff
@@ -189,7 +189,7 @@
         const language = getBrowserLanguage();
 
         // Determine nameValue and typeValue based on sanitized inputs
-        const nameValue = zaakValue || taakValue;
+        const nameValue = taakValue || zaakValue;
```

Comment:
Hmm. What issues does this reversal solve? Not sure I get it..

Reply:
When document is added to a task, 'taak' AND 'zaak' query vars are both in, but task should be mentioned. So let then 'taak' prevail. Should I add this explanation in the code @edgarvonk ?

Reply:
Added comment